src/main/java/edu/ucsb/cs156/happiercows/entities/Commons.java

```diff
@@ -29,6 +29,7 @@
   private double milkPrice;
   private double startingBalance;
   private LocalDateTime startingDate;
+  private double degradationRate;
```

Comment:
Please consider also adding this field into the fixtures for Commons and UserCommons so that front end developers can test their code:

https://github.com/ucsb-cs156-s22/s22-5pm-happycows/blob/main/frontend/src/fixtures/commonsFixtures.js

https://github.com/ucsb-cs156-s22/s22-5pm-happycows/blob/main/frontend/src/fixtures/userCommonsFixtures.js

In general the fixtures should be kept up to date with what the JSON that the backend returns will look like.  There may be other changes that need to be made there too.

The good news is that this will increase the point value of the PR!

Reply:
Added the degradationRate field to the appropriate fixtures.